tests/integration/learning-plan-banner/mulk-learning-plan-banner.spec.ts

```
@@ -0,0 +1,132 @@
+const imageLink = (page: Page) => banner(page).locator('a').first();
+const anyCTAInBanner = (page: Page) => banner(page).locator(`a[href="${LP_URL}"]`).first();
+
+async function scrollToBottom(page: Page) {
+  await page.evaluate(() => window.scrollTo(0, document.documentElement.scrollHeight));
+}
+
+async function waitForBannerPresent(page: Page) {
+  await scrollToBottom(page);
+  // Wait for virtualized list to settle
+  await page.waitForLoadState('domcontentloaded');
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)